apps/docs/src/guide/contracts/deploying-contracts.md

```diff
@@ -9,6 +9,8 @@
 
 This guide walks you through deploying a contract using the SDK, covering loading contract artifacts, initializing a contract factory, and deploying the contract.
 
+> **Note:** The maximum contract deployment size supported by the SDK is 100 KB. The SDK will `throw` for contracts larger than this size.
```

Comment:
We could link the error code :)

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2796/commits/11b41c25020a3165b8f7286d473625a93f4b3412

Reply:
My only hesitation there is we use `forc deploy` for that guide, which won't have the same restrictions

Reply:
Agreed. What about here?
 - [guide/fuels-cli/commands.html#fuels-deploy](https://fuels-5gz5is123-fuel-labs.vercel.app/guide/fuels-cli/commands.html#fuels-deploy)

Reply:
Good point https://github.com/FuelLabs/fuels-ts/pull/2796/commits/b9992eb9edd8f26d1f8022a45933a9d816ed383b

Reply:
Nit, sorry - but should we combine both notes at the top?

```
Note

1. We recommend using the [..]
2. When using `fuels deploy` locally, the maximum contract deployment size [..]
```

---

![deploy](https://github.com/user-attachments/assets/87319aa8-974c-41a8-8d7e-76d03b6c375e)

Reply:
Thoughts? I had spread them out a bit because they seemed a bit messy, but actually not too bad together.

<img width="727" alt="Screenshot 2024-07-19 at 14 18 29" src="https://github.com/user-attachments/assets/9f8b7243-8951-4746-b613-418c7fa48ba9">